app/javascript/components/rooms/RoomCard.jsx

```diff
@@ -32,10 +32,13 @@ export default function RoomCard({ room }) {
 
         <Stack className="my-4">
           <Card.Title className="mb-0"> { room.name } </Card.Title>
-          { room.shared_owner ? (
+          { room.shared_owner && (
             <span className="text-muted">{ t('room.shared_by', { room }) } </span>
+          )}
+          { room.last_session ? (
+            <span className="text-muted"> Last Session: { room.last_session } </span>
```

Comment:
If it's feasible, can we add in localization here?

Reply:
Done